storage/google/cloud/storage/client.py

```diff
@@ -202,7 +202,7 @@ def batch(self):
         """
         return Batch(client=self)
 
-    def get_bucket(self, bucket_name):
+    def get_bucket(self, bucket_or_name):
```

Comment:
I'd prefer this to be a separate PR, too, but it's okay this once. In general a PR should correspond to 1 easily-described change (new feature / bug fix). Now this PR has 2 new features (but they are related, at least).

Reply:
Took this file out of review - please disregard, thanks!